### DISCUSSION TITLE
Sharing Conditions

### DISCUSSION THREAD

**Discussion by sorinciupitu** [immich-app/immich#4261] [Feature Request]
### The feature

It is possible to add sharing conditions on shared albums? Example: I have an account, my wife another account. If I take pictures with my daughter and assign face to her, I want to be able to make an album with all pictures where is detected my daughter face and share that album with my wife. Also when my wife take pictures with our daughter, that pictures must automatically added to the shared album so I can view them.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1673
